dashboard/app/serializers/api/v1/pd/district_report_data_table_serializer.rb

```diff
@@ -0,0 +1,65 @@
+class Api::V1::Pd::DistrictReportDataTableSerializer < ::Api::V1::DataTableSerializerBase
+  attributes :cols, :rows
+
+  def cols
+    names = [
+      'District Name',
+      'Workshop Organizer',
+      'Workshop Organizer Id',
+      'Facilitators',
+      'Workshop Dates',
+      'Workshop Type',
+      'Course',
+      'Subject',
+      'School',
+      'Teacher Name',
+      'Teacher Id',
+      'Teacher Email',
+      'Year',
+      {label: 'Hours', type: 'number'},
+      {label: 'Days', type: 'number'}
+    ]
+    if scope.admin?
+      names += [
+       'Payment Type',
+       {label: 'Payment Rate', type: 'number'},
+       {label: 'Qualified', type: 'boolean'},
+       {label: 'Payment Amount', type: 'number'}
+      ]
+    end
+
+    data_table_columns names
+  end
+
+  def rows
+    object.map do |row|
+      keys = [
```

Comment:
oic. You mean the row name and column names. Yeah I suppose they are a translation, at least in the common case. Rows use a different format for non-string types (see below) than columns do, but I should be able to clean this up so it's a single definition. I'll take a look.